### ISSUE TITLE
Standardize placeholder text & update font colours

### ISSUE THREAD

**Issue by NicoleGovvy** [bcgov/cas-registration#3154]
#### Describe the task

Review & update the font colours from the BC gov design system for placeholder and read-only text in BCIERS as current colours do not meet accessibility standards.

https://www2.gov.bc.ca/gov/content/digital/design-system/foundations/colour#palettes

![Image](https://github.com/user-attachments/assets/b0f24f2d-3ab8-44f8-9d4d-b19882c62143)

### **Acceptance criteria - Placeholder text colour**

Change text color to #9f9d9c on all placeholder form fields in:

**CAS pages:**

Administration tile
- [ ] Administration > Operators: search box placeholder text
- [ ] Administration > Operations: search box placeholder text
- [ ] Administration > Operations > View facilities: search box placeholder text
- [ ] Administration > Operator administrator and access requests: search box placeholder text
- [ ] Administration > Contacts: search box placeholder text

Transfers tile
- [ ] Transfers: search box placeholder text

Reporting tile
- [ ] 

### **Acceptance criteria - Read-only radio buttons appear selectable**

Disabled Checkboxes and radio buttons appear clickable, users try to edit a non-editable field.
- [ ] Change selection color to #474543 and background to #EDEBE9

screenshot below which shows the correct colors 
![Image](https://github.com/user-attachments/assets/317cc7ed-04e0-4f30-9385-76c38796caf3)

Disabled/read-only multiselect dropdown chips are not legible
- [ ] Change text color to #474543 and chip background to #EDEBE9

register operation form
Change text color to #474543 and background to #EDEBE9 on all disabled form fields in:

reporting form

register operation form
Operation type placeholder text is black instead of gray

change to #9f9d9c

**Comment by NicoleGovvy** [comment]
Email from the BC Gov Design System group about values & formatting. **Note placeholder text is NOT recommended for accessibility issues.**

There are two different tokens with the same value (#9F9D9C) in the DS, which is probably confusing things somewhat:
 
typography.color.placeholder: this is a legacy token that was intended to be removed in a future release*
typography.color.disabled: this is explicitly intended for text that is disabled and non-interactable
 
*before we had to pause work on the DS
 
For “placeholder and read-only/inactive” cases, I’d generally suggest using typography.color.secondary (#474543) instead.
 
In our input components in the design system, we generally avoided and advised against using placeholder/ghost text inside inputs, in favour of explicit label and description fields. The rationale here is that placeholder text [can cause additional usability issues](https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.w3.org%2FWAI%2Ftutorials%2Fforms%2Finstructions%2F%23placeholder-text&data=05%7C02%7CNicole.1.Smith%40gov.bc.ca%7Cafa225d1b84b4a3707b108dd67d93de0%7C6fdb52003d0d4a8ab036d3685e359adc%7C0%7C0%7C638780904915132364%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=fSa9NjF80c7jedXcRvxY0M0GkiNcLfJd4rbGEg2ijdg%3D&reserved=0).
 
Here’s [the DS TextField](https://designsystem.gov.bc.ca/react-components/?path=/story/components-text-inputs-text-field-single-line--text-field-template) to illustrate how we handled disabled and read-only text:
 
‘Disabled’ case

Semantically labelled with [disabled](https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdeveloper.mozilla.org%2Fen-US%2Fdocs%2FWeb%2FHTML%2FAttributes%2Fdisabled&data=05%7C02%7CNicole.1.Smith%40gov.bc.ca%7Cafa225d1b84b4a3707b108dd67d93de0%7C6fdb52003d0d4a8ab036d3685e359adc%7C0%7C0%7C638780904915155662%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=NfFsT87KXGOCwLtCLvp4LfscMWOxcdv6muHACvdgGv4%3D&reserved=0) / [aria-disabled](https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdeveloper.mozilla.org%2Fen-US%2Fdocs%2FWeb%2FAccessibility%2FARIA%2FReference%2FAttributes%2Faria-disabled&data=05%7C02%7CNicole.1.Smith%40gov.bc.ca%7Cafa225d1b84b4a3707b108dd67d93de0%7C6fdb52003d0d4a8ab036d3685e359adc%7C0%7C0%7C638780904915168465%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=PEvELldPoebFEkv9tNGIkjh00nYaGZl1imK31X%2Byi5g%3D&reserved=0) attributes
Grey fill and not-allowed cursor effect provide visual hints for sighted users
Input is not focusable or text-selectable
Exempt from WCAG SC 1.4.3 because it is an inactive UI element
 
 
‘Read-only’ case

Semantically labeled with [readonly](https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdeveloper.mozilla.org%2Fen-US%2Fdocs%2FWeb%2FHTML%2FAttributes%2Freadonly&data=05%7C02%7CNicole.1.Smith%40gov.bc.ca%7Cafa225d1b84b4a3707b108dd67d93de0%7C6fdb52003d0d4a8ab036d3685e359adc%7C0%7C0%7C638780904915180299%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=zVuVUibGI3NY8kasQuqI59Y5OMGOdOJVjkhJT%2BAoEE0%3D&reserved=0) attribute
Grey fill provides visual hint
Input can be focused and text can be selected
Meets SC 1.4.3 contrast requirement
 
Hope that helps!
 
Thanks,
Marcus
